### PR TITLE
Arrow aside collections

### PR DESCRIPTION
**Pull request by VeenaYemmiganur**
# Description
Integration of Arrow to Story templates - Passing Template Config / Aside Collection
Fixes # (issue-reference)
https://github.com/quintype/ace-planning/issues/734

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "fontfaceobserver": "^2.1.0",
     "intersection-observer": "^0.12.2",
     "lodash": "^4.17.21",
+    "node-fetch": "^3.3.0",
```
Comment (Jeevan-Kishore) [inline package.json:24]: do we need this? we have axios & wretch

File: app/api/utils.js [hunk 2]
```diff
@@ -29,3 +29,18 @@ export const getAuthorStories = async (id, offset, limit) => {
   ).json();
   return items;
 };
+
+export function loadRelatedStories(story, config) {
+  const { mountAt } = config || {};
+  const mountAtPrefix = mountAt || "";
+  const FIELDS =
```
Comment (Jeevan-Kishore) [inline app/api/utils.js:36]: add in constants.js file

**Review by Jeevan-Kishore — CHANGES_REQUESTED** [review]

**Review by Jeevan-Kishore — APPROVED** [review]

File: app/isomorphic/components/story-templates/listicle-story/index.js [hunk 3]
```diff
@@ -3,9 +3,15 @@ import ListicleStoryTemplate from "../../../arrow/components/Rows/StoryTemplates
...
-  const templateSpecific = {};
-  // { templateType: "headline-hero-priority", authorDetails: { template: "centerAligned" } };
-
+  const templateSpecific = {
+    templateType: "headline-sideway",
+    showSection: true,
+    publishedDetails: {
+      enablePublishedTime: true,
+      enableUpdatedTime: true,
+      showReadTime: true,
```
Comment (Jeevan-Kishore) [inline app/isomorphic/components/story-templates/listicle-story/index.js:12]: these lines seem to repeat, can we have a function which sends out the config and override as needed using parameters to the same function?

Comment (Jeevan-Kishore) [reply app/isomorphic/components/story-templates/listicle-story/index.js:12]:
```suggestion
const getTemplateConfig = (object) => {{
    templateType: "headline-sideway",
    showSection: true,
    publishedDetails: {
      enablePublishedTime: true,
      enableUpdatedTime: true,
      showReadTime: true,
      }
      }, ...object```

**Review by Jeevan-Kishore — CHANGES_REQUESTED** [review]

**Review by Jeevan-Kishore — APPROVED** [review]

**Review by cricket-lover — APPROVED** [review]

**Pull request merged**